business/workloads.go

```diff
@@ -99,10 +99,12 @@ type LogEntry struct {
 
 // LogOptions holds query parameter values
 type LogOptions struct {
-	Duration *time.Duration
-	IsProxy  bool // fetching logs for Istio Proxy (Envoy access log)
-	MaxLines *int
+	Duration  *time.Duration
+	IsProxy   bool // fetching logs for Istio Proxy (Envoy access log)
+	IsZtunnel bool
```

Comment:
This might be better as something like an "enum" now e.g. `LogType = "Proxy"` or something.

Reply:
Updated